umh-core/cmd/renderer_test.go

```
@@ -0,0 +1,119 @@
+		}}
+		doc := renderSupervisorChildrenYAML(credentialed, placeholderUUID)
+		Expect(childNames(doc)).To(ContainElement("persistence"))
+		Expect(childNames(doc)).To(ContainElement("communicator"))
+
+		var parsed decodedChildren
+		Expect(yaml.Unmarshal([]byte(doc), &parsed)).To(Succeed())
+		var comm *decodedChild
+		for i := range parsed.Children {
+			if parsed.Children[i].Name == "communicator" {
+				comm = &parsed.Children[i]
+				break
```

golangci-lint (2.12.2)

[error] break with no blank line before

(nlreturn)